frontend/src/Components/Exercise.js

```diff
@@ -47,8 +62,11 @@ const Exercise = ({selected}) => {
                             variant='flushed' 
                             mx={2} 
                             onChange={typingHandler} 
-                            value={inputVal}/>
-                            <Button w={'70px'} onClick={()=>{onTry()}}>Try</Button>
+                            value={inputVal}
+                            placeholder={`E.g. ${selected.inputExample}`}
```

Comment:
added a placeholder to make it easier to understand how a correct input would look